src/utilities/FormRUtilities.tsx

```diff
@@ -117,6 +119,17 @@ export class FormRUtilities {
     return value ? value : message;
   }
 
+  public static dispaySubmissionDate(date: DateType) {
+    const message = (
+      <h3>
+        {"Form Submitted on: " +
+          DateUtilities.ToLocalDate(formData.submissionDate)}
```

Comment:
... the `date` parameter goes inside the `ToLocalDate` method rather than the argument (the actual value passed when we use the util method in the components).